cmd/eudico/main.go

```diff
@@ -23,6 +23,7 @@ var eudCmds = []*cli.Command{
 	lcli.WithCategory("daemon", tpowCmd),
 	lcli.WithCategory("daemon", filcnsCmd),
 	lcli.WithCategory("daemon", mirCmd),
+	lcli.WithCategory("daemon", genesisCmd),
```

Comment:
I don't think we should use it into the `daemon` category, right? It probably would be better if we add a `genesis` cmd/category for this.

Reply:
The command to run is:
```
./eudico genesis new --subnet-id /root/t01 --out genesis.car
```